### ISSUE TITLE
illegal hardware instruction  swift build

### ISSUE THREAD

**Issue by kiliankoe** [tid-kijyun/Kanna#108]
Hi, I'm currently trying to install Kanna via SPM. First thing SPM fails at is the missing name in the Package.swift, see #107. 
After doing that however all I get is the following right after SPM resolves the correct version. Any clue what this could be about?

```
HEAD is now at 63b013b Add name to Package.swift
Resolved version: 2.1.0
[1]    84128 illegal hardware instruction  swift build
```

(the version is due to the fact that I tagged my PR commit in my fork, so that I can reference the tag via SPM)
### Installation method:

SPM
### Library version:
- [x] v2.0.0
### Xcode version:

n/a

**Comment by broadwaylamb** [comment]
If you want to use SPM, you can try this fork of Kanna: https://github.com/WeirdMath/Scrape

**Comment by kiliankoe** [comment]
Will definitely give that a spin, thanks for the link!

**Comment by tid-kijyun** [comment]
Hi @broadwaylamb 

Thank you for working hard. 👍

I've just released a new version(2.0.1) that supports Linux and CSS selector.
This is a beta version, but it should work.

If you have time, please try it.

### Create the workspace
```
$ mkdir sample
$ cd sample
```

### Initialize the workspace
```
$ swift package init --type=executable
```

### Edit Package.swift
```
$ vi Package.swift

import PackageDescription

let package = Package(
    name: "sample",
    
    dependencies: [
        .Package(url: "https://github.com/tid-kijyun/Kanna.git", majorVersion: 2)
    ]
)
```

### Build packages
```
$ swift build -Xcc -I/usr/include/libxml2
```

### Edit main.swift
```
$ vi Sources/main.swift

import Kanna

if let doc = HTML(html: "<div>Hello spm! <span class='hoge'>Hello Linux!</span></div>", encoding: .utf8) {
    print(doc.css("div").first?.text)
    print(doc.css(".hoge").first?.text)
}
```

### Build & Run
```
$ swift build && ./.build/debug/sample
```

output:
```
Optional("Hello spm! Hello Linux!")
Optional("Hello Linux!")
```

Thanks

**Comment by broadwaylamb** [comment]
Cool! I'll definitely check it out.

**Comment by kiliankoe** [comment]
I'm not quite sure how I'm running into this, but on running `swift build -Xcc -I/usr/include/libxml2`, I get a lot of errors referring to undeclared types and unresolved identifiers. I'm sure this is an issue on my end... Any idea how this might be happening? 

[Pastebin link](http://pastebin.com/raw/s3tZVxDe), so I'm not spamming here as much.

**Comment by tid-kijyun** [comment]
Did you install the `libxml2-dev`?
Please try install it.


### Example for Ubuntu
```
$ sudo apt-get install libxml2
$ sudo apt-get install libxml2-dev

// build again
$ swift build -Xcc -I/usr/include/libxml2
```
If you are using other environment, please install in each method.

**Comment by tid-kijyun** [comment]
Installation by SPM is available on Mac now. It was available only in linux until version 2.0.1. 

```
swift build -Xswiftc -DSPM
```
